src/components/dataTable/checkbox.scss

```diff
@@ -0,0 +1,10 @@
+table.md-table {
+
+  th, td {
+    md-checkbox {
+      width: 20px;
```

Comment:
Use variables here as well. 

Also hide checkbox label and align it self in the center (as done in the list as well)